content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,276 @@
+Management guide, so now we just call `dvc exp run` to run the experiment.
+
+We now commit the result to git (and push it to our git remote) and the new
+model will show up in the model registry in Studio. We will also push the
+resulting artifact to our DVC Remote by calling `dvc push`.
+
+<details id="push-click-to-see-how-artifacts-are-registered">
+
+#### 💡 Expand to see how adding models to the registry works under the hood
+
+When we call the `log_artifact()` method, DVC takes all the information we
+provided in the call and edits the `dvc.yaml` file which will now contain the
+following lines
+
+```yaml
+artifacts:
+  pool-segmentation:
+    path: models/model.pkl
+    type: model
+    desc:
+      This is a Computer Vision (CV) model that's segmenting out swimming pools
+      from satellite images.
+    labels:
+      - cv
+      - segmentation
+      - satellite-images
+```
+
+When you push the resulting file to your git remote it is parsed by Studio and
+it then shows all model artifacts from your `dvc.yaml` files in the model
+registry.
+
+</details>
+
+You should see something like the following picture in the Studio Model
+Registry.
+
+![Newly added model in the Model Registry](/img/mr-newly-added-model.png)
+
+To get acquainted with how DVC stores and shares data, see our
+[Get Started guide on Data Versioning](/doc/start/data-management/data-versioning).
+
+<details id="push-click-to-see-other-ways-to-add-models">
+
+#### 💡 Expand to see other ways to add models
+
+The other two options are to use the Studio's graphical user interface to add
+models interactively or to manually edit `dvc.yaml` files to add information
+about model artifacts. To get more details on the ways to add models have a look
+at the
+[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
+
+</details>
+
+## DVC Model registry overview
+
+In DVC Studio we can access the model registry by clicking on Models in the top
+bar menu. This will show us a dashboard with all models from all projects we
+have access to.
+
+From the dashboard we will have an overview of all models, latest model versions
+as well stages each of the model versions is assigned to. We can get more
+details for each model by clicking on the model name.
+
+<admon type="tip" id="GTO-tip">
+
+The DVC Model registry uses the [GTO library](/doc/gto) to manage model versions
+and model lifecycle stages by particularly formatted git tags.
+
+<<<<<<< Updated upstream To see how it all works and how you can use GTO
+directly without the Studio UI, you can explore the "under the hood" expandable
+content in this guide! ======= To see how it all works and how you can use GTO
+directly without the Studio fUI, you can explore the "under the hood" expandable
+content in this guide!
+
+> > > > > > > Stashed changes
+
+</admon>
+
+<details id="under-the-hood-model-registry">
+
+#### 💡 Expand to see how the model registry works under the hood
```

Comment:
Need some kind of intro to GTO here

Reply:
I added a one sentence description to the tip above the details